res/ktopt_moveit_parameters.yaml

```diff
@@ -4,6 +4,11 @@ ktopt_interface:
     description: "Robot description to be loaded by the internal Drake MultibodyPlant",
     default_value: "package://drake_models/franka_description/urdf/panda_arm_hand.urdf",
   }
+  link_ee: {
```

Comment:
Why do we need this parameter? The constraint message itself contains a link name field and we should likely use that:

https://github.com/moveit/moveit_msgs/blob/ros2/msg/PositionConstraint.msg#L5-L6

Reply:
I put this because I did not see a panda_EE frame on the visualised robot_description on RViz. Whereas the Drake urdf has it, and the PositionConstraint requires this. I'll debug this further.

Reply:
Hmmm, weird. Can you use another frame that does exist in both URDFs?

Or maybe this is sufficient to have us make a local version of the Drake URDF? Worth discussing in 30 min.

Reply:
Latest commit reflects this.